### PR TITLE
Updates Form API

### PR DESCRIPTION
**Pull request by jobannon**
Updates Form API to include `selfServiceUser`

### CODE REVIEW

File: site/docs/v1/tech/apis/_form-request-body.adoc [hunk 1]
```diff
@@ -45,6 +45,7 @@ The type of form being created, a form type cannot be changed after the form has
  * `registration` - This form will be used for self service registration.
  * `adminRegistration` - This form be used to customize the add and edit User Registration form in the FusionAuth UI. [since]#Available since 1.20.0#
```
Comment (mooreds) [inline site/docs/v1/tech/apis/_form-request-body.adoc:46]: Should these be in alphabetical order?

File: site/docs/v1/tech/apis/_form-response-body.adoc [hunk 2]
```diff
@@ -28,6 +28,7 @@ The form type. The possible values are:
  * `registration` - This form will be used for self service registration.
  * `adminRegistration` - This form be used to customize the add and edit User Registration form in the FusionAuth UI. [since]#Available since 1.20.0#
```
Comment (mooreds) [inline site/docs/v1/tech/apis/_form-response-body.adoc:29]: Can you correct this? 'This form be used'

File: site/docs/v1/tech/apis/_forms-response-body.adoc [hunk 3]
```diff
@@ -31,6 +31,7 @@ The Form type. The possible values are:
  * `registration` - This form will be used for self service registration.
  * `adminRegistration` - This form be used to customize the add and edit User Registration form in the FusionAuth UI. [since]#Available since 1.20.0#
  * `adminUser` - This form can be used to customize the add and edit User form in the FusionAuth UI. [since]#Available since 1.20.0#
```
Comment (mooreds) [inline site/docs/v1/tech/apis/_forms-response-body.adoc:33]: Can we sync up 'can be' and 'will be' here? Across all the pages?

**Review by mooreds — CHANGES_REQUESTED** [review]
A few changes needed.

File: site/docs/v1/tech/apis/_form-request-body.adoc (hunk 1 above)
Comment (jobannon) [reply site/docs/v1/tech/apis/_form-request-body.adoc:46]: They are in enum order, I think.  Probably good to reorder.

**Pull request merged**